specs/deneb/light-client/sync-protocol.md

```diff
@@ -65,9 +65,11 @@ def is_valid_light_client_header(header: LightClientHeader) -> bool:
     epoch = compute_epoch_at_slot(header.beacon.slot)
 
     # [New in Deneb:EIP4844]
-    if epoch < DENEB_FORK_EPOCH:
-        if header.execution.blob_gas_used != uint64(0) or header.execution.excess_blob_gas != uint64(0):
-            return False
+    if epoch < config.DENEB_FORK_EPOCH:
+        return (
+            header.execution.blob_gas_used == uint64(0)
+            and header.execution.excess_blob_gas == uint64(0)
+        )
```

Comment:
I don't think this change is correct. It doesn't behave the same. For pre-Deneb, with unset blob gas fields, it will no longer check if the execution branch is valid. It will simply return true now.